packages/arb-token-bridge-ui/tests/e2e/specs/importToken.cy.ts

```diff
@@ -0,0 +1,200 @@
+import {
+  ERC20TokenAddressL1,
+  ERC20TokenAddressL2,
+  invalidTokenAddress,
+  resetSeenTimeStampCache
+} from '../../support/common'
+
+const importTokenThroughUI = (address: string) => {
+  // Click on the ETH dropdown (Select token button)
+  cy.findByRole('button', { name: 'Select Token' })
+    .should('be.visible')
+    .should('have.text', 'ETH')
+    .click({ scrollBehavior: false })
+
+  // open the Select Token popup
+  return cy
+    .findByPlaceholderText(/Search by token name/i)
+    .should('be.visible')
+    .type(address, { scrollBehavior: false })
+    .then(() => {
+      // Click on the Add new token button
+      cy.findByRole('button', { name: 'Add New Token' })
+        .should('be.visible')
+        .click({ scrollBehavior: false })
+    })
+}
+
+describe('Import token', () => {
+  context('User import token through UI', () => {
```

Comment:
the tests in this file cover most cases for the token search panel

if we consider all functionalities of the token search panel,
we'd want to include test cases for when users type the token name to search
but technically it isn't "import token" (the token is probably present on one of the lists)

what do you say if we cover all actions relevant to token search in this test file?

Reply:
Do you have other cases in mind? I can imagine `Add` button grayed if input is invalid (empty, less or more than 42 char)

Reply:
also the token not found case

and search by symbol

Reply:
Not found case is handled [here](https://github.com/OffchainLabs/arb-token-bridge/pull/502/files#diff-3b66b55eaeaa0524e7363bc7ab04058744eb53f337a7779da9dd0efb3f229ecbR93). I'll add test for the symbol search